src/python/test/test_nextflow.py

```diff
@@ -103,11 +115,27 @@ def test_src_profile(self):
     @unittest.skipIf(USING_PYTHON2,
         'Skipping as the Nextflow template from which applets are built is for Py3 interpreter only')
     def test_src_inputs(self):
```

Comment:
Would it be possible to add a docstring to this test that described what it tests. So what in the future we know what's going on here, for example """ Tests that the --input= ... was passed to the nextflow run command in the applet source code. This happens because .. """ Whatever will make it easier to read the code and understand our motivations in the future. Thanks a bunch.

Reply:
Done. Please help take a look, thanks! @commandlinegirl

Reply:
Yes, this is great. We should also have it documented as a part of official documentation